tracing-appender/src/file_appender.rs

```diff
@@ -0,0 +1,109 @@
+use crate::inner::BufWriterFactory;
+use crate::worker::Worker;
+use crate::Rotation;
+use chrono::Utc;
+use crossbeam_channel::{bounded, Sender};
+use std::sync::atomic::{AtomicU64, Ordering};
+use std::{io, thread};
+use tracing_subscriber::fmt::MakeWriter;
+
+#[allow(dead_code)]
+pub struct FileAppender {
+    log_writer: FileWriter,
+    worker_thread: thread::JoinHandle<()>,
+}
+
+impl FileAppender {
```

Comment:
We probably want a public `new` constructor as well as a builder, that constructs a default configuration. This could be something like
```rust
pub fn new(log_directory: &str, log_filename_prefix: &str) -> Self {
    FileAppenderBuilder::default().build(log_directory, log_filename_prefix, ...)
}
```

We can keep the private constructor but rename it to something else.